SU2_CFD/src/solvers/CSolver.cpp

```diff
@@ -3316,29 +3316,23 @@ void CSolver::LoadInletProfile(CGeometry **geometry,
   string Interpolation_Function, Interpolation_Type;
   bool Interpolate = false;
 
-  su2double *Normal = new su2double[nDim];
-
   unsigned long Marker_Counter = 0;
 
-  bool turbulent = (config->GetKind_Solver() == RANS ||
-                    config->GetKind_Solver() == INC_RANS ||
-                    config->GetKind_Solver() == ADJ_RANS ||
-                    config->GetKind_Solver() == DISC_ADJ_RANS ||
-                    config->GetKind_Solver() == DISC_ADJ_INC_RANS);
-
   unsigned short nVar_Turb = 0;
-  if (turbulent)
```

Comment:
yes! Sanity checks should be (and are in this case) done in cconfig, so we can safely assume that the solver is a turbulence solver.